unittests/auth_tests.cpp

```diff
@@ -362,9 +362,7 @@ try {
    BOOST_CHECK_EXCEPTION(chain.create_account(name("aaaaaaaaaaaaa")), action_validate_exception,
                          fc_exception_message_is("account names can only be 12 chars long"));
 
-   // Creating a new account with non-privileged account, should fail
-   BOOST_CHECK_EXCEPTION(chain.create_account(name("dandy.joe"), name("joe")), action_validate_exception,
-                         fc_exception_message_is("Only privileged accounts can create new accounts"));
+   // Note: Only privileged accounts can create accounts enforced by sysio.system contract
```

Comment:
assuming we switch back to enforcing this where it was originally, can we add this assert back in?